src/app/technical-record/tyres/tyres.component.spec.ts

```diff
@@ -1,64 +1,67 @@
-import {HttpClientTestingModule} from '@angular/common/http/testing';
-import {MatDialogModule} from '@angular/material/dialog';
-import {FormsModule, ReactiveFormsModule} from '@angular/forms';
-import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
-import {MaterialModule} from '@app/material.module';
-import {SharedModule} from '@app/shared/shared.module';
-import {RouterTestingModule} from '@angular/router/testing';
-import {FontAwesomeModule} from '@fortawesome/angular-fontawesome';
-import {NgrxFormsModule} from 'ngrx-forms';
-import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
-import {VehicleSummaryComponent} from '@app/technical-record/vehicle-summary/vehicle-summary.component';
-import {TechnicalRecordComponent} from '@app/technical-record/technical-record.component';
+import { async, ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
+import { TechRecordHelpersService } from '@app/technical-record/tech-record-helpers.service';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { TyresComponent } from '@app/technical-record/tyres/tyres.component';
+import { SharedModule } from '@app/shared/shared.module';
+import { RouterTestingModule } from '@angular/router/testing';
 
-describe('VehicleSummaryComponent', () => {
-
-  let component: VehicleSummaryComponent;
-  let fixture: ComponentFixture<VehicleSummaryComponent>;
+describe('TyresComponent', () => {
+  let component: TyresComponent;
+  let fixture: ComponentFixture<TyresComponent>;
   let injector: TestBed;
 
   beforeEach(() => {
-
     TestBed.configureTestingModule({
-      imports: [
-        StoreModule.forRoot(appReducers),
-        HttpClientTestingModule,
-        MatDialogModule,
-        FormsModule,
-        ReactiveFormsModule,
-        BrowserAnimationsModule,
-        MaterialModule,
-        SharedModule,
-        RouterTestingModule,
-        FontAwesomeModule,
-        ReactiveFormsModule,
-        NgrxFormsModule
-      ],
-      declarations: [VehicleSummaryComponent, TechnicalRecordComponent],
-      providers: [
-        TechRecordHelpersService
-      ],
+      imports: [SharedModule, RouterTestingModule],
+      declarations: [TyresComponent],
+      providers: [TechRecordHelpersService],
       schemas: [CUSTOM_ELEMENTS_SCHEMA]
-
     }).compileComponents();
 
-    fixture = TestBed.createComponent(VehicleSummaryComponent);
+    fixture = TestBed.createComponent(TyresComponent);
     injector = getTestBed();
     component = fixture.componentInstance;
     component.activeRecord = {
-      'vin': 'XMGDE02FS0H012345',
-      'vehicleSize': 'small',
-      'testStationName': 'Rowe, Wunsch and Wisoky',
-      'vehicleId': 'JY58FPP',
-      'vehicleType': 'psv',
-      'axles': [
-        { 'parkingBrakeMrk': false, 'axleNumber': 1 },
-        { 'parkingBrakeMrk': true, 'axleNumber': 2 },
-        { 'parkingBrakeMrk': false, 'axleNumber': 3 }
+      vin: 'XMGDE02FS0H012345',
```

Comment:
we should have a smaller mock object, Within each test partially mutate the key you want to update. I can explain better